common/app/model/content/Atoms.scala

```diff
@@ -288,6 +288,50 @@ object InteractiveAtom {
 
 object RecipeAtom {
   def make(atom: AtomApiAtom): RecipeAtom = RecipeAtom(atom.id, atom, atom.data.asInstanceOf[AtomData.Recipe].recipe)
+
+  def picture(r: RecipeAtom): Option[model.ImageMedia] = {
+    r.data.images.headOption.map{ img => MediaAtom.imageMediaMake(img, "")}
+  }
+
+  def totalTime(recipe: RecipeAtom): Option[Int] = {
+    (recipe.data.time.preparation ++ recipe.data.time.cooking).map(_.toInt).reduceOption(_ + _)
+  }
+
+  def yieldServingType(serves: com.gu.contentatom.thrift.atom.recipe.Serves): String = {
+    serves.`type` match {
+      case "serves" => "servings"
+      case "makes" => s"${serves.unit}"
+      case "quantity" => "portions"
+    }
+  }
+
+  def ingredientValue(ingredient: com.gu.contentatom.thrift.atom.recipe.Ingredient): String = {
```

Comment:
I would call that `formatIngredient` to be consistent with the other function names.